index.html

```diff
@@ -156,9 +182,8 @@
 <p>
 <div>  
 You can also help us keep the Hackerspace running by donating <strong>Bitcoin</strong> to the
-<span class="mono"><a href="bitcoin:1386nJBU5xV5gdR9pMAea2wRSz6rz4apFu?amount=0.01&label=Thanks%20for%20supporting%20the%20Hackerspace%21">1386nJBU5xV5gdR9pMAea2wRSz6rz4apFu</a></span> address! Thank you!<br> <img src="/img/HackerspaceSupport.png">
+<span class="mono"><a href="bitcoin:1386nJBU5xV5gdR9pMAea2wRSz6rz4apFu?amount=0.01&amp;label=Thanks%20for%20supporting%20the%20Hackerspace%21">1386nJBU5xV5gdR9pMAea2wRSz6rz4apFu</a></span> address! Thank you!<br> <img src="img/HackerspaceSupport.png" alt="Hackerspace support Bitcoin QR code">
 </div>
-</p>
```

Comment:
Checked it out, it's because having `<div>` inside `<p>` is invalid, and the validator fixes it by assuming the first `<p>` is just itself and nothing inside (and optional no end tag), then  a `<div>`, then finds an end tag, and it screams. The correct way is rearranging the hierarchy, and removing the `<p>` altogether. Should do a rewrite for the rest of the page too.

Updated it in https://github.com/taipeihackerspace/web/pull/4 see what do you think.